web-src/src/main-app/store/scriptConfig.js

```diff
@@ -333,8 +334,14 @@ function reconnect(state, internalState, commit, dispatch, selectedScript) {
             logError(error);
 
             if (error instanceof SocketClosedError) {
+              
+                if (error.code === 422) {
+                  commit('SET_ERROR', `${CANNOT_PARSE_ERROR_PREFIX} "${selectedScript}"`);
+                  return;
+                }
```

Comment:
I would suggest to remove custom handling here. Script server can start sending 422 for other cases as well, so always showing "parsing error" is not good. We should always rely on `error.reason`, which is provided by the server